x-pack/plugins/observability_solution/observability_onboarding/public/application/quickstart_flows/auto_detect/supported_integrations_list.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiBadge,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiTextColor,
+  EuiToolTip,
+  IconType,
+  useEuiTheme,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import apacheIconSrc from '../../../assets/apache.svg';
+import dockerIconSrc from '../../../assets/docker.svg';
+import nginxIconSrc from '../../../assets/nginx.svg';
+import mysqlIconSrc from '../../../assets/mysql.svg';
+
+const SUPPORTED_INTEGRATIONS_LIST = [
+  'Apache',
+  'Docker',
+  'Nginx',
+  'System',
+  'MySQL',
+  'PostgreSQL',
+  'Redis',
+  'Haproxy',
+  'Kafka',
+  'RabbitMQ',
+  'Prometheus',
+  'Apache Tomcat',
+  'MongoDB',
+] as const;
+
+type SupportedIntegrationName = (typeof SUPPORTED_INTEGRATIONS_LIST)[number];
+
+interface SupportedIntegrationItem {
+  title: SupportedIntegrationName;
+  icon: IconType;
+}
+
+const FEATURED_INTEGRATIONS_LIST: SupportedIntegrationItem[] = [
+  { title: 'Apache', icon: apacheIconSrc },
+  { title: 'Docker', icon: dockerIconSrc },
+  { title: 'Nginx', icon: nginxIconSrc },
+  { title: 'MySQL', icon: mysqlIconSrc },
+  { title: 'System', icon: 'desktop' },
+];
+
+export function SupportedIntegrationsList() {
+  const {
+    euiTheme: { colors },
+  } = useEuiTheme();
+  const customLogFilesTitle = i18n.translate(
+    'xpack.observability_onboarding.autoDetectPanel.supportedIntegrationsList.customIntegrationTitle',
+    { defaultMessage: 'Custom .log files' }
+  );
+  return (
+    <EuiFlexGroup gutterSize="s" responsive={false} css={{ flexWrap: 'wrap' }}>
+      {FEATURED_INTEGRATIONS_LIST.map(({ title, icon }) => (
+        <EuiFlexItem grow={false}>
+          <EuiBadge iconType={icon} color="hollow" key={title}>
+            {title}
+          </EuiBadge>
+        </EuiFlexItem>
+      ))}
+
+      <EuiBadge iconType="documents" color="hollow">
+        {customLogFilesTitle}
+      </EuiBadge>
+
+      <EuiToolTip
+        content={
+          <EuiText size="s">
+            <ul>
+              {SUPPORTED_INTEGRATIONS_LIST.map((integration) => (
```

Comment:
Shouldn't the tooltip only list the ones that are not already spelled out individually?

Reply:
I took this from [the desings](https://www.figma.com/design/CPhMyRNOgo0wsEiaIMZJ14/Onboarding-Quick-Starts?node-id=1022-101201&t=5tvnrPIOkfg7xAJp-1). At first I thought the same, that it should have only the remaining services, but it seems convenient to have the full list in one place, I suspect that was the logic behind the designs.

Reply:
Ah, no worries then :)